packages/sui-studio/bin/sui-studio-init.js

```diff
@@ -0,0 +1,174 @@
+/* eslint no-console:0 */
+
+const colors = require('colors')
+const program = require('commander')
+const BASE_DIR = process.cwd()
+const fse = require('fs-extra')
+
+program
+  .on('--help', () => {
+    console.log('  Examples:')
+    console.log('')
+    console.log('    $ suistudio init <project-name>')
+    console.log('    $ suistudio init sui-studio')
+    console.log('    $ suistudio init --help')
+    console.log('    $ suistudio init -h')
+    console.log('')
+  })
+  .parse(process.argv)
+
+const [PROJECT_NAME] = program.args
+
+const showError = (msg) => {
+  program.outputHelp(txt => colors.red(txt))
+  console.error(colors.red(msg))
+  process.exit(1)
+}
+
+const writeFile = (path, body) => {
+  fse.outputFile(
+    path,
+    body,
+    err => {
+      if (err) { showError(`Fail creating ${path}`) }
+      console.log(colors.gray(`Created ${path}`))
+    }
+  )
+}
+
+const createDir = (path) => {
+  fse.mkdirp(path, err => {
+    if (err) { showError(`Fail creating ${path}`) }
+    console.log(colors.gray(`Created ${path}`))
+  })
+}
+
+if (!PROJECT_NAME) { showError('the project name must be defined') }
+
+createDir(`${BASE_DIR}/${PROJECT_NAME}/components`)
+createDir(`${BASE_DIR}/${PROJECT_NAME}/demo`)
+
+writeFile(
+`${BASE_DIR}/${PROJECT_NAME}/components/README.md`,
+'# Here put a description about your project'
+)
+
+writeFile(
+`${BASE_DIR}/${PROJECT_NAME}/package.json`,
+`{
+  "name": "${PROJECT_NAME}",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "phoenix": "rm -Rf node_modules && npm it",
+    "deploy": "suistudio build && surge public/ -d ${PROJECT_NAME}.surge.sh",
+    "co": "git-cz",
+    "lint:js": "node_modules/.bin/eslint --ext .js,.jsx src/",
+    "lint:sass": "node_modules/.bin/sass-lint src/**/*.scss -c -v",
+    "lint": "npm run lint:js && npm run lint:sass"
+  },
+  "repository": {
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@schibstedspain/suistudio-fatigue-dev": "github:SUI-Components/suistudio-fatigue-dev"
+  },
+  "peerDependencies": {
+    "react": "15",
+    "react-dom": "15"
+  },
+  "dependencies": {},
+  "config": {
+    "ghooks": {
```

Comment:
The init command will be moved to a new sui-studio-create.
sui-studio will be executed locally and sui-studio-create globally. I'll refactor in sui-studio-create directly.